### PR TITLE
About-myself and works section

### PR DESCRIPTION
**Pull request by Cisco-the-wayword**
in this PR, I created the following: 
- Add works section
- Add the about-myself section
- add button validations.
- Add resume link

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
+          <h4 class="post">Multi-Post Stories Gain+Glory</h4>
+          <div class="tags">
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+          </div>
+          <div class="button-space">
```
Comment (fobadara) [inline index.html:130]: [REQUIRED] One of the purposes of this task is to train you to follow client requirements as much as possible. Kindleremove the spaces for all the work cards pointed out n the image below: 

|Your Design|
|-----------|
|image|

File: index.html [hunk 2]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
+            <button type="button" class="project-text">See Project</button>
+          </div>
```
Comment (fobadara) [inline index.html:149]: [REQUIRED] One of the purposes of this task is to train you to follow client requirements as much as possible. Kindleremove the spaces for all the work cards pointed out n the image below: 

|Your Design|
|-----------|
|image|

File: index.html [hunk 3]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
+        <div class="gray-area"></div>
+        <div class="description">
+          <h4 class="post">Multi-Post Stories Gain+Glory</h4>
+          <div class="tags">
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
```
Comment (fobadara) [inline index.html:162]: [REQUIRED] One of the purposes of this task is to train you to follow client requirements as much as possible. Kindleremove the spaces for all the work cards pointed out n the image below: 

|Your Design|
|-----------|
|image|

File: index.html [hunk 4]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
+        </div>
+      </div>
+      <div class="block">
+        <div class="gray-area"></div>
+        <div class="description">
+          <h4 class="post">Multi-Post Stories Gain+Glory</h4>
+          <div class="tags">
+            <ul>
+              <li>Ruby on rails</li>
+              <li>css</li>
```
Comment (fobadara) [inline index.html:176]: [REQUIRED] One of the purposes of this task is to train you to follow client requirements as much as possible. Kindleremove the spaces for all the work cards pointed out n the image below: 

|Your Design|
|-----------|
|image|

File: index.html [hunk 5]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
+              <li>JavScript</li>
+              <li>html</li>
+            </ul>
+          </div>
+          <div class="button-space">
+            <button type="button" class="project-text">See Project</button>
```
Comment (fobadara) [inline index.html:199]: [REQUIRED] One of the purposes of this task is to train you to follow client requirements as much as possible. Kindleremove the spaces for all the work cards pointed out n the image below: 

|Your Design|
|-----------|
|image|

File: index.html [hunk 6]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
```
Comment (fobadara) [inline index.html:114]: [REQUIRED] Kindly ensure that your button matches the template in all states (pressed, active and enabled). Kindly make sure that it still has the prescribed border-radius and when the button is clicked the colour should change back immediately after the click is released not when a new click on something else.

|Your Design|
|-----------|
|image|

|Template's button colour indifferent states|
|----------------------------------------|
|image|

File: index.html [hunk 7]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
```
Comment (fobadara) [inline index.html:148]: [REQUIRED] Kindly ensure that your button matches the template in all states (pressed, active and enabled). Kindly make sure that it still has the prescribed border-radius and when the button is clicked the colour should change back immediately after the click is released not when a new click on something else.

|Your Design|
|-----------|
|image|

|Template's button colour indifferent states|
|----------------------------------------|
|image|

File: index.html [hunk 8]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
```
Comment (fobadara) [inline index.html:165]: [REQUIRED] Kindly ensure that your button matches the template in all states (pressed, active and enabled). Kindly make sure that it still has the prescribed border-radius and when the button is clicked the colour should change back immediately after the click is released not when a new click on something else.

|Your Design|
|-----------|
|image|

|Template's button colour indifferent states|
|----------------------------------------|
|image|

File: index.html [hunk 9]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
```
Comment (fobadara) [inline index.html:182]: [REQUIRED] Kindly ensure that your button matches the template in all states (pressed, active and enabled). Kindly make sure that it still has the prescribed border-radius and when the button is clicked the colour should change back immediately after the click is released not when a new click on something else.

|Your Design|
|-----------|
|image|

|Template's button colour indifferent states|
|----------------------------------------|
|image|

File: index.html (hunk 5 above)
Comment (fobadara) [inline index.html:199]: [REQUIRED] Kindly ensure that your button matches the template in all states (pressed, active and enabled). Kindly make sure that it still has the prescribed border-radius and when the button is clicked the colour should change back immediately after the click is released not when a new click on something else.

|Your Design|
|-----------|
|image|

|Template's button colour indifferent states|
|----------------------------------------|
|image|

File: index.html [hunk 10]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
+        <h2 class="about-head">About me</h2>
+        <div>
+          <p class="about-text">
+            I can help you build a product, feature or website. Look through
+            some of my work and experience! If you like what you see and have a
+            project you need developed, don't hesitate to contact me.
+          </p>
+        </div>
+        <div class="resume-block">
+          <button type="button" class="resume-text">Get My Resume</button>
```
Comment (fobadara) [inline index.html:216]: [REQUIRED] Kindly ensure that your button matches the template in all states (pressed, active and enabled). Kindly make sure that it still has the prescribed border-radius and when the button is clicked the colour should change back immediately after the click is released not when a new click on something else.

|Your Design|
|-----------|
|image|

|Template's button colour indifferent states|
|----------------------------------------|
|image|

File: index.html [hunk 11]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
+        />
+        <h2 class="about-head">About me</h2>
+        <div>
+          <p class="about-text">
+            I can help you build a product, feature or website. Look through
+            some of my work and experience! If you like what you see and have a
+            project you need developed, don't hesitate to contact me.
+          </p>
+        </div>
+        <div class="resume-block">
```
Comment (fobadara) [inline index.html:215]: [REQUIRED] kindly ensure the background image is pressing towards the right and left wall without spaces. Look at the space wit the red curves.

|Your Design|
|-----------|
|image|

File: index.html [hunk 12]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
+        <div>
+          <img
+            class="drop-down"
+            src="Images/drop-down.png"
+            alt="drop-down button"
+          />
+        </div>
+      </section>
+    </main>
+    <section class="rest">
```
Comment (fobadara) [inline index.html:95]: [OPTIONAL] I think this section should be within your main tag. The idea would be that:
- All your code apart from nav and footer will be in the main tag
- Within the main ag you have different section tags
- One section will be for the landing page
- Another section for recent works
- And another section for about me

File: index.html (hunk 6 above)
Comment (fobadara) [inline index.html]: [REQUIRED] You are expected to use grid to arrange the cards in the works section while you use flexbox to arrange the contents of each card but I don't see you using grid

File: index.html [hunk 13]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
+      <h1 class="works">My Recent Works</h1>
```
Comment (fobadara) [inline index.html]: [REQUIRED] for accessibility you are expected to use the `h1` tag only once in a document. Kindly change this to another heading tag

File: index.html [hunk 14]
```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
...
+      </div>
+    </section>
+    <div class="the-rest">
+      <section class="about">
+        <div class="gap"></div>
+        <img
+          src="Images/illustration.png"
+          alt="illustration"
+          class="illustration"
+        />
```
Comment (fobadara) [inline index.html]: [REQUIRED] Kindly ensure all images that are just for decoration use CSS background property not `img` tags

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Cisco-the-wayword, 

Good job so far!
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

You can also consider:
- [OPTIONAL] This is not a requirement for this milestone but I think you should fix it

|Your Landing Page|
|--------------|
|image|


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 11 above)
Comment (Cisco-the-wayword) [reply index.html:215]: I used up the full recommended width

File: styles.css [hunk 15]
```diff
@@ -132,3 +133,424 @@ ul {
...
+
+.the-rest {
+  padding-left: 25px;
+}
+
+.about {
+  background-image: url("Images/Aboutme-background\ \(1\).png"), url("Images/illustration.png");
+  background-position: left top, right top;
+  background-repeat: no-repeat;
+}
```
Comment (Santiago220991) [inline styles.css:339]: I think you did a great job with the changes suggested by the last code reviewer. :+1:  However there are still some things to improve. The background should fit also on the left side of the container as suggested last code reviewer. :eyes: 
image

File: styles.css [hunk 16]
```diff
@@ -132,3 +133,424 @@ ul {
...
+.container {
+  height: 352px;
+  background: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-bottom: 24px;
+}
```
Comment (Santiago220991) [inline styles.css]: Kindly make sure when you fix the background image in the about section the borders should be still present in the technologies cards. :eyes: 
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi @Cisco-the-wayword , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
:heavy_check_mark: No linter errors.
:heavy_check_mark: Followed GitHub flow.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly make the mobile version should be responsive until the desktop screen size. :+1:  :smiley: :smiley: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @Cisco-the-wayword, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and I wish you a pleasant learning experience at Microverse! 🎉 

congratulations


## Good Highlights✔

- Linter checks are passing
- Good commit messages
- You have implemented requested changes


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**